public/app/core/components/AppChrome/AppChrome.tsx

```diff
@@ -32,6 +32,9 @@ export function AppChrome({ children }: Props) {
   const dockedMenuBreakpoint = theme.breakpoints.values.xl;
   const dockedMenuLocalStorageState = store.getBool(DOCKED_LOCAL_STORAGE_KEY, true);
   const menuDockedAndOpen = !state.chromeless && state.megaMenuDocked && state.megaMenuOpen;
+  // TODO how to get this state from the scene?
+  const scopeDashboardsState = true;
```

Comment:
You can simply write a `useScopesDashboardsState` React Hook in `app/features/scopes` and export it in `app/features/scopes/index.ts` (we specifically try to avoid exporting the singleton instances of the scenes).

Something in the lines of:
```ts
import { scopesDashboardsScene } from './instance';

export const useScopesDashboardsState = () => {
  return scopesDashboardsScene?.useState() ?? {}; // fallback to empty object to avoid issues when destructuring
}
```

Then you can just consume it here as `const { isPanelOpened } = useScopesDashboardsState()`;

Reply:
nice, thanks @bfmatei 🙇